Makefile

```diff
@@ -44,6 +44,11 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/quality.txt requirements/quality.in
 	pip-compile --upgrade -o requirements/test.txt requirements/base.in requirements/test.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
+	# Workaround for packages that break pip-tools due to python version specifiers somewhere upstream
+	# See https://github.com/nvie/pip-tools/issues/206
+	cat requirements/dev-extra.txt >> requirements/dev.txt
```

Comment:
I don't remember running into this when I used the cookie-cutter for a modulestore repo.
Is there a JIRA ticket for addressing this?